build/testBuildImages.sh

```diff
@@ -43,6 +43,7 @@ export MSBUILDDEBUGPATH="$msbuildDebugLogsDir"
 export MSBUILDDISABLENODEREUSE=1
 
 dotnet test \
+    --blame \
```

Comment:
On a recent test run, my build failed with the error `The active test run was aborted. Reason: Test host process crashed`, so adding the following flag to see if we can get more information:

![image](https://user-images.githubusercontent.com/6559784/66866460-33611a00-ef4e-11e9-9e5c-e366a23af365.png)